Tchap/target.yml

```diff
@@ -145,20 +153,33 @@ targets:
     - path: ../Riot/Modules/KeyBackup
     - path: ../Riot/Modules/KeyVerification
     - path: ../Riot/Modules/MediaPicker
+    - path: ../Riot/Modules/PublicRoomList
```

Comment:
Why do you add this resource?

Reply:
This resource is not needed for this PR. I removed it in last commit.